diff --git a/lib/dea/responders/dea_locator.rb b/lib/dea/responders/dea_locator.rb
index 97ec014..688fa21 100644
--- a/lib/dea/responders/dea_locator.rb
+++ b/lib/dea/responders/dea_locator.rb
@@ -53,7 +53,7 @@ module Dea::Responders
     # keep track of all deas that it can use to run apps
     def start_periodic_dea_advertise
       advertise_interval = config["intervals"]["advertise"] || DEFAULT_ADVERTISE_INTERVAL
-      @advertise_timer = EM.add_periodic_timer(advertise_interval) { advertise }
+      # @advertise_timer = EM.add_periodic_timer(advertise_interval) { advertise }
     end

     def stop_periodic_dea_advertise
