diff --git a/lib/dea/bootstrap.rb b/lib/dea/bootstrap.rb
index 3d25b17..b7cd1ad 100644
--- a/lib/dea/bootstrap.rb
+++ b/lib/dea/bootstrap.rb
@@ -308,7 +308,7 @@ module Dea

     def start_finish
       nats.publish("dea.start", Dea::Protocol::V1::HelloMessage.generate(self))
-      locator_responders.map(&:advertise)
+      # locator_responders.map(&:advertise)
     end

     def register_directory_server_v2
